.github/workflows/ci.yml

```diff
@@ -10,8 +10,8 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['25', '26', '27']
-        rebar: ['3.24']
+        otp: ['26', '27', '28']
+        rebar: ['3.25']
```

Comment:
I'd consider moving to 26+, but I don't wanna make that decision in this pull request. For now I think it's enough if we test for those (26+).